presto-elasticsearch/src/test/java/com/facebook/presto/elasticsearch/TestPasswordAuthentication.java

```diff
@@ -94,12 +94,12 @@ public void test()
                 .build());
 
         performRequest(
-                        "POST",
-                        "/test/_doc?refresh",
-                        ImmutableMap.of(),
-                        new NStringEntity(json, ContentType.APPLICATION_JSON),
-                        client,
-                        new BasicHeader("Authorization", format("Basic %s", Base64.encodeAsString(format("%s:%s", USER, PASSWORD).getBytes(StandardCharsets.UTF_8)))));
+                "POST",
+                "/test/_doc?refresh",
+                ImmutableMap.of(),
+                new NStringEntity(json, ContentType.APPLICATION_JSON),
+                client,
+                new BasicHeader("Authorization", format("Basic %s", Base64.getEncoder().encodeToString(format("%s:%s", USER, PASSWORD).getBytes(StandardCharsets.UTF_8)))));
```

Comment:
nit: let's revert the indentation changes here as well and make only the required code change